packages/sync-actions/test/category-sync.spec.js

```diff
@@ -75,6 +75,93 @@ describe('Actions', () => {
     expect(actual).toEqual(expected)
   })
 
+  it('build `setCustomType` action on category with no `custom` object', () => {
```

Comment:
All of these its have the `build `setCustomType` action on category` part in common. Thus we could wrap all of them in a describe block:
```js
describe('changing the custom type of a category', () => {
  describe('existing category has no `custom` object', () => {
    it('should build a `setCustomType` action with the new type')
  })
  describe('existing category has an empty `custom` object', () => {
    it('should build a `setCustomType` action with the new type')
  })
  describe('existing category has a `custom` object', () => {
    describe('new category has no `custom` object', () => {
      it('should build a `setCustomType` action to unset the `custom` type')
    })
    describe('new category has an empty `custom` object', () => {
      it('should build a `setCustomType` action to unset the `custom` type')
    })
  })
})
```